pkgs/tools/system/ps_mem/default.nix

```
@@ -1,16 +1,16 @@
 { stdenv, pythonPackages, fetchFromGitHub }:
 
 let
-  version = "3.12";
+  version = "3.13";
   pname = "ps_mem";
 in pythonPackages.buildPythonApplication {
   name = "${pname}-${version}";
 
   src = fetchFromGitHub {
     owner = "pixelb";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "0kcxlmfisbwf24p2k72njfyfp22fjr9p9zalg9b4w0yhnlzk24ph";
+    rev = "9f54e1aa3a87ec176ce8b71f02673e0d8293b344";
```

if you're not pulling from an official release, then the convention is to mark the version as "unstable-YYYY-MM-DD" of the commit.

---

@jonringer For now I've removed this update and asked ps_mem author to mark it as release if it indeed was so https://github.com/pixelb/ps_mem/issues/51

---

You could also leave a comment such as
```suggestion
    # determined by pypi upload date
    rev = "9f54e1aa3a87ec176ce8b71f02673e0d8293b344";
```

---

@jonringer No, no. Pypi still points to 3.12 version. There is no hurry to update ps_mem. I just borrowed that update from Gentoo ebuild. But I think it just better if program author will make a proper release

---

@jonringer ps_mem author tagged release. I've added update commit